test/chunk.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const assert = require('assert')
+const { chunk } = require('../index')
+
+describe('chunk', () => {
+    it('should return an empty array when input is null', () => {
+        assert.deepStrictEqual(chunk(null, 1), [])
```

Comment:
@Kumar-Kishan can we make it fail when inputs are invalid? It could make the behaviour odd and future devs might depend on that

Reply:
so an error.